.github/workflows/release-repo-dispatch.yml

```diff
@@ -0,0 +1,18 @@
+name: release-repo-dispatch
+
+on:
+  repository_dispatch:
+
+jobs:
+  update_version_from_repository_dispatch:
+    uses: killbill/gh-actions-shared/.github/workflows/update-version.yml@main
+
+  # If CI failed, should we revert commit? I don't think so, since we still have release.yml with "workflow_dispatch",
```

Comment:
I think we should run CI in parallel. It takes a very long time + as you said, we can fix things manually (this would be in other repos and wouldn't be a release bliocker).

Reply:
"parallel" means we don't need to call `needs: update_version_from_repository_dispatch` so each job will run together? But we need to update version before we run build and release, no?